Numpy fails to compile on MSVC 14 (VS2015)

On the master branch of numpy, line 432 of arraytypes.c.src hosts the following line
```
memset(ov + datalen, 0, (PyArray_DESCR(ap)->elsize - datalen));
```
Which leads to the following compiler error:
```
numpy\core\src\multiarray\arraytypes.c.src(432): error C2036: 'void *': unknown size
```
casting ov to a char* will allow compilation to continue:

```
memset((char*)ov + datalen, 0, (PyArray_DESCR(ap)->elsize - datalen));
```

--- a/numpy/core/src/multiarray/arraytypes.c.src
+++ b/numpy/core/src/multiarray/arraytypes.c.src
@@ -429,7 +429,7 @@ UNICODE_setitem(PyObject *op, void *ov, void *vap)
 #endif
     /* Fill in the rest of the space with 0 */
     if (PyArray_DESCR(ap)->elsize > datalen) {
-        memset(ov + datalen, 0, (PyArray_DESCR(ap)->elsize - datalen));
+        memset((char*)ov + datalen, 0, (PyArray_DESCR(ap)->elsize - datalen));
     }
     if (!PyArray_ISNOTSWAPPED(ap)) {
         byte_swap_vector(ov, PyArray_DESCR(ap)->elsize >> 2, 4);

